panther_analysis_tool/backend/lambda_client.py

```diff
@@ -110,7 +110,11 @@ def bulk_upload(self, params: BulkUploadParams) -> BackendResponse[BulkUploadRes
 
         if backend_response_failed(resp):
             err = BackendError(resp.data)
+
             err.permanent = True
+            if "another upload" in resp.data.get("body", ""):
```

Comment:
Can we pluck this logic into a shared function? Whenever we have to do a string check like this in more than one place I think its a good idea to keep that logic in one spot

Reply:
agreed will update

Reply:
updated, let me know if that is what you had in mind